signin-style.css

```
@@ -0,0 +1,112 @@
+html {
+  font-family: "Noto Sans KR", sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 16px auto;
+  width: 360px;
+  background-color: #f0f6ff;
+}
+
+form {
+  margin: 150px auto 16px;
+  width: 360px;
+}
+
+label {
+  color: #111322;
+  display: block;
+  margin: 16px 0 8px;
+}
+
+input {
+  border: 1px solid #9fa6b2;
+  border-radius: 10px;
+  color: #111322;
+  display: block;
+  font-size: 16px;
+  line-height: 24px;
+  margin: 8px 0;
+  padding: 16px 24px;
+  width: 100%;
+}
+
+.login-button {
+  background: linear-gradient(to right,#6d6afe,#6ADFFE);
```

```suggestion
  background: linear-gradient(to right, #6d6afe, #6ADFFE);
```

사이사이 약간의 공백 어떨까요~~?!